tools/nightly_checkout.py

```diff
@@ -0,0 +1,252 @@
+def checkout_nightly_version(spdir):
+    """Get's the nightly version and then checks it out."""
+    # first get the git version from the installed module
+    version_fname = os.path.join(spdir, "torch", "version.py")
+    with open(version_fname) as f:
+        lines = f.read().splitlines()
+    for line in lines:
+        if not line.startswith('git_version'):
+            continue
+        git_version = literal_eval(line.partition("=")[2].strip())
+        break
+    else:
+        raise RuntimeError(f"Could not find git_version in {version_fname}")
+    print(f"Found released git version {git_version}")
+    # now cross refernce with nightly version
+    _ensure_commit(git_version)
+    cmd = ["git", "show", "--no-patch", "--format=%s", git_version]
+    p = subprocess.run(cmd, capture_output=True, check=True, text=True)
+    m = SHA1_RE.search(p.stdout)
+    if m is None:
+        raise RuntimeError(f"Could not find nightly release in git history:\n  {p.stdout}")
+    nightly_version = m.group(1)
+    print(f"Found nightly release version {nightly_version}")
+    # now checkout nightly version
+    _ensure_commit(nightly_version)
+    cmd = ["git", "checkout", nightly_version]
+    p = subprocess.run(cmd, check=True)
+
+
+def _get_listing_linux(source_dir):
+    listing = glob.glob(os.path.join(source_dir, "*.so"))
+    listing.append(os.path.join(source_dir, "version.py"))
+    listing.extend(glob.glob(os.path.join(source_dir, "lib", "*.so")))
+    listing.append(os.path.join(source_dir, "bin"))
+    return listing
+
+
+def _get_listing_osx(source_dir):
+    # oddly, these are .so files even on Mac
+    listing = glob.glob(os.path.join(source_dir, "*.so"))
+    listing.append(os.path.join(source_dir, "version.py"))
+    listing.extend(glob.glob(os.path.join(source_dir, "lib", "*.dylib")))
+    listing.append(os.path.join(source_dir, "bin"))
+    return listing
+
+
+def _get_listing_win(source_dir):
+    listing = glob.glob(os.path.join(source_dir, "*.pyd"))
+    listing.append(os.path.join(source_dir, "version.py"))
+    listing.extend(glob.glob(os.path.join(source_dir, "lib", "*.lib")))
+    listing.extend(glob.glob(os.path.join(source_dir, "lib", "*.dll")))
+    listing.append(os.path.join(source_dir, "bin"))
+    return listing
+
+
+def _get_listing(source_dir, platform):
+    if platform.startswith("linux"):
+        listing = _get_listing_linux(source_dir)
+    elif platform.startswith("osx"):
+        listing = _get_listing_osx(source_dir)
+    elif platform.startswith("win"):
+        listing = _get_listing_win(source_dir)
+    else:
+        raise RuntimeError(f"Platform {platform!r} not recognized")
+    return listing
+
+
+def _remove_existing(trg, is_dir):
+    if os.path.exists(trg):
+        if is_dir:
+            shutil.rmtree(trg)
+        else:
+            os.remove(trg)
+
+def _move_single(src, target_dir, mover, verb):
+    is_dir = os.path.isdir(src)
+    base = os.path.basename(src)
```

Comment:
This removes relative path information, like if the file was in the `lib` directory. Possible fix: https://github.com/Quansight/conda-pytorch/pull/3/files

Reply:
Yeah, sorry I fixed this on my own!